casks/tunnel-client-app.rb

```diff
@@ -0,0 +1,10 @@
+cask 'tunnel-client-app' do
+  version '1.0.0'
+  sha256 'e92eb93d152de457054658b603bafed7e45f97c8d92f14db5cb959a210c6bb38'
+
+  url 'https://storage.googleapis.com/tunnel-client-app-binaries/TunnelClient.app.tar.gz'
```

Comment:
@BlakeMesdag I told Nick it was fine not to use a signed URL here. There's nothing secret stored in https://github.com/Shopify/tunnel-client-app

Reply:
Here's a log of the error using a signed URL:
```shell
$ brew cask install tunnel-client-app
==> Satisfying dependencies
==> Downloading https://storage.googleapis.com/tunnel-client-app-binaries/TunnelClient.app.tar.gz?GoogleAccessId=oauth-tunnel-client@data-148903.iam.gserviceaccount.com&Expires=151
Error: Download failed on Cask 'tunnel-client-app' with message: File name too long @ rb_sysopen - /usr/local/var/homebrew/locks/tunnel-client-app--1.0.com&Expires=1519771459&Signature=ZVI7Q43JFIWLgPq7M72vS0esCodZsQ6i0FXpLjdmH7QpXF8%2FxzENqgBRafsVlM13Rmu2QbwgTYxoXAi%2BjiJOQbnU2fAfm%2FUojCnfta0KJY8F2%2F3I9hdUUgPvspkZ9Ugi9V3JXZhbEHEOWCKoBnWFYsjOLejmYiIdNZ9mUNePxGb6pIYpUqORPx0v9Lj1mBUl05DeGW06LoHxCLxwGdMrNPiAXnKTgjz9ZhdSrPDT9w9Y%2F1M6eAREs3TBcx42nV41TYemwlySi%2FitIzPQ7FLdmf%2BWPe3wo3iRxFxgGjbbYYRlcTM9TdssbVY0HTFJMyzHlYXRYkMZ3qKl648Pj%2BGKNA%3D%3D.incomplete.lock
```